Dockerfile

```diff
@@ -1,34 +1,44 @@
 # build stage
-FROM golang:1-alpine AS build
-
-RUN apk --no-cache add git gcc make musl-dev curl bash openssh-client
+FROM golang:1 AS build
 
 ENV \
   STRONGBOX_VERSION=master \
   KUSTOMIZE_VERSION=v4.5.5
 
-RUN os=$(go env GOOS) && arch=$(go env GOARCH) \
-  && curl -Ls https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_${os}_${arch}.tar.gz \
-    | tar xz -C /usr/local/bin/ \
-  && chmod +x /usr/local/bin/kustomize \
+RUN go install sigs.k8s.io/kustomize/kustomize/v4@${KUSTOMIZE_VERSION} \
```

Comment:
Good for development, but for production you probably want to fix versions and control upgrades, otherwise you can get caught out with a broken master.

Reply:
kustomize is actually versioned, strongbox is on master. I will fix it